sqlalchemy_filterset/operators.py

```diff
@@ -1,7 +1,22 @@
+from typing import Any
+
 from sqlalchemy.sql import ColumnElement
+from sqlalchemy.sql import operators as sa_op
 
 from sqlalchemy_filterset.types import ModelAttribute
 
 
 def icontains(field: ModelAttribute, value: str) -> ColumnElement:
     return field.ilike(f"%{value}%")
+
+
+def is_null(field: ModelAttribute, value: bool) -> Any:
+    if not isinstance(value, bool):
```

Comment:
Validation of input parameters is not the target of the library, type-hint is enough.
Let's make it a little bit simpler.
```python
if value:
    return sa_op.is_(field, None)
else: 
    return sa_op.is_not(field, None)
```

Reply:
Thanks for the feedback! Just fixed it